test/v200/utils/common/command_test_utils.go

```diff
@@ -0,0 +1,212 @@
+package common
+
+import (
+	"fmt"
+
+	schema "github.com/devfile/api/v2/pkg/apis/workspaces/v1alpha2"
+)
+
+// commandAdded adds a new command to the test schema data and notifies the follower
+func (testDevFile *TestDevfile) commandAdded(command schema.Command) {
+	LogInfoMessage(fmt.Sprintf("command added Id: %s", command.Id))
+	testDevFile.SchemaDevFile.Commands = append(testDevFile.SchemaDevFile.Commands, command)
+	if testDevFile.Follower != nil {
+		testDevFile.Follower.AddCommand(command)
+	}
+}
+
+// commandUpdated and notifies the follower of the command which has been updated
```

Comment:
more descriptive comment here pls, probably as added func to remain consistent

Reply:
Will remove surplus "and" in next PR. This is not quite the same as the add function as the add function also updates the schema.

Reply:
Fixed in new commit